sdk/imagebuilder/arm-imagebuilder/CHANGELOG.md

```diff
@@ -1,15 +1,54 @@
 # Release History
+    
+## 3.0.0 (2023-03-24)
+    
+**Features**
 
-## 2.1.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+  - Added operation group Triggers
+  - Added Interface DistributeVersioner
+  - Added Interface DistributeVersionerLatest
+  - Added Interface DistributeVersionerSource
+  - Added Interface ImageTemplateFileValidator
+  - Added Interface ImageTemplatePropertiesOptimize
+  - Added Interface ImageTemplatePropertiesOptimizeVmBoot
+  - Added Interface SourceImageTriggerProperties
+  - Added Interface TargetRegion
+  - Added Interface Trigger
+  - Added Interface TriggerCollection
+  - Added Interface TriggerProperties
+  - Added Interface TriggersCreateOrUpdateOptionalParams
+  - Added Interface TriggersDeleteHeaders
+  - Added Interface TriggersDeleteOptionalParams
+  - Added Interface TriggersGetOptionalParams
+  - Added Interface TriggersListByImageTemplateNextOptionalParams
+  - Added Interface TriggersListByImageTemplateOptionalParams
+  - Added Interface TriggerStatus
+  - Added Interface UserAssignedIdentity
+  - Added Interface VirtualMachineImageTemplatesDeleteHeaders
+  - Added Type Alias DistributeVersionerUnion
+  - Added Type Alias TriggerPropertiesUnion
+  - Added Type Alias TriggersCreateOrUpdateResponse
+  - Added Type Alias TriggersGetResponse
+  - Added Type Alias TriggersListByImageTemplateNextResponse
+  - Added Type Alias TriggersListByImageTemplateResponse
+  - Added Type Alias VMBootOptimizationState
+  - Interface ImageTemplate has a new optional parameter optimize
+  - Interface ImageTemplateSharedImageDistributor has a new optional parameter targetRegions
+  - Interface ImageTemplateSharedImageDistributor has a new optional parameter versioning
+  - Interface ImageTemplateSharedImageVersionSource has a new optional parameter exactVersion
+  - Interface ImageTemplateVhdDistributor has a new optional parameter uri
+  - Type of parameter type of interface ImageTemplateInVMValidator is changed from "Shell" | "PowerShell" to "Shell" | "PowerShell" | "File"
+  - Enum KnownSharedImageStorageAccountType has a new value PremiumLRS
 
-### Other Changes
+**Breaking Changes**
 
+  - Type of parameter userAssignedIdentities of interface ImageTemplateIdentity is changed from {
```

Comment:
this breaking may caused by：https://github.com/Azure/azure-rest-api-specs/pull/22901